module13and14/LoopCanvas/LoopCanvas/Models/Block.swift

```diff
@@ -0,0 +1,179 @@
+//
+//  BlockModel.swift
+//  LoopCanvas
+//
+//  Created by Peter Rice on 6/8/24.
+//
+
+import Foundation
+import SwiftUI
+
+class Block: ObservableObject, Identifiable, Codable {
+  // Persistent props
+  @Published var id: Int
```

Comment:
It might be cleaner to have a single structure that is published and then have the `id`, `location`, `color`, `visible` and anything else that needs to be published jsut in that structure. It'll make the code a little more wordy, but as you are adding things to be published it'll be easier to organize them.